tools/close_by_signature.py

```diff
@@ -0,0 +1,243 @@
+import sys
+import tempfile
+import logging
+import argparse
+import pprint
+
+from add_triage_signature import (
+    config_logger, get_credentials, get_jira_client, get_issues,
+    SIGNATURES,
+)
+
+
+logger = logging.getLogger(__name__)
+
+dry_run_stdout = None
```

Comment:
Let's not make it global in this module
We can easily pass it around in the 3 functions here